virt-v2v/cold/entrypoint.go

```diff
@@ -98,9 +98,17 @@ func buildCommand() []string {
 	}
 
 	fmt.Println("Preparing virt-v2v")
+
+	if checkEnvVariablesSet("V2V_RootDisk") {
+		rootDisk := fmt.Sprintf("/dev/%s", os.Getenv("V2V_RootDisk"))
+		virtV2vArgs = append(virtV2vArgs, "--root", rootDisk)
+	} else {
+		virtV2vArgs = append(virtV2vArgs, "--root", "first")
+	}
```

Comment:
ok, this way it also applies to OVA but is it intentional? (it's relevant also for OVA but did you make sure nothing breaks this way?)
and please just determine the root disk (<device>/first) and append the argument in a single place

Reply:
From the tests I've did I saw its applied for ova as well, I'm not sure if its really doing anything but from virt-v2v at least its seams to be supported.

Reply:
@ahadas I've tested it for OVA and its not really considering the input so I returned it to vsphere only.